packages/charts/src/components/RadialChart/index.tsx

```diff
@@ -66,23 +66,29 @@ export class RadialChart extends PureComponent<RadialChartPropTypes> {
       options
     );
 
+    const styles = {
```

Comment:
Maybe another name for that object or pass it inline? It shadows the styles object which gets passed to withStyles.

Reply:
is adjusted